core/src/main/java/com/mercadopago/sdk/android/core/di/KoinModuleProvider.kt

```diff
@@ -0,0 +1,39 @@
+package com.mercadopago.sdk.android.core.di
+
+import org.koin.core.Koin
+import org.koin.core.module.Module
+import org.koin.core.logger.Level
+import org.koin.dsl.koinApplication
+
+/**
+ * Interface that`s provide the module list
+ */
+interface CoreKoinModuleProvider {
```

Comment:
since some of those classes can't be internal, we can limit the usage outside our project
```suggestion
@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
interface CoreKoinModuleProvider {
```